deps/build.jl

```diff
@@ -13,8 +13,10 @@ else
     const libfftwf_name = "libfftw3f_threads"
 end
 
-libfftw = library_dependency(libfftw_name, aliases=[replace(libfftw_name, "3", ""), libfftw_name * "-3"])
-libfftwf = library_dependency(libfftwf_name, aliases=[replace(libfftwf_name, "3", ""), libfftwf_name * "-3"])
+libfftw = library_dependency(libfftw_name, aliases=[replace(libfftw_name, "3", ""),
+                                                    replace(libfftw_name, "3", "-3")])
```

Comment:
see https://build.opensuse.org/package/view_file/windows:mingw:win64/mingw64-fftw3/mingw64-fftw3.spec?expand=1
it's actually `libfftw3-3.dll` and `libfftw3f-3.dll` according to that spec file